.github/workflows/ci.yml

```diff
@@ -9,19 +9,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10.x, 12.x]
+        node: [12.x]
         os: [ubuntu-latest, macOS-latest, windows-latest]
     steps:
-      - uses: actions/checkout@v1
+      - name: Checkout
+        uses: actions/checkout@v1
+
       - name: Use Node.js ${{ matrix.node }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node }}
           registry-url: https://registry.npmjs.org
+
       - name: Install dependencies
```

Comment:
It's helpful to add a step that prints out the exact `npm` and `node` version

Reply:
Why would you need this to be printed as it's already returned by the job?

![image](https://user-images.githubusercontent.com/237229/81971609-58566200-95ef-11ea-9d69-1106eab5d67f.png)

Reply:
Oh this is new to me 😮  
GitHub may add it by default now ❤